src/hearings/containers/request-hearing/welsh-hearing/welsh-hearing.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ErrorMessage } from 'src/app/models';
+
+export const NAME_ERROR: ErrorMessage = {
+  title: 'There is a problem',
+  description: 'You must select Yes or No',
+  fieldId: 'welsh-hint',
+};
+
+@Component({
+  selector: 'exui-welsh-hearing',
+  templateUrl: './welsh-hearing.component.html',
+  styleUrls: ['./welsh-hearing.component.scss'],
+})
+export class WelshHearingComponent implements OnInit {
+  public error: ErrorMessage = null;
+  public welshForm: FormGroup;
+
+  constructor(private readonly formBuilder: FormBuilder) { }
+
+  public ngOnInit() {
```

Comment:
could you add void as return type please?

Reply:
updated with void return type